### ISSUE TITLE
Error in exported KML File

### ISSUE THREAD

**Issue by s-moon** [findsorguk/findsorguk#1096]
### Expected behaviour
When exporting a KML file, it should provide a fully compliant XML file.

### Actual behaviour
In some instances, it has text like the following at the top:
```
<b>Notice</b>:  Undefined variable: description in <b>/var/www/finds.org.uk/app/modules/database/views/scripts/ajax/kml.phtml</b> on line <b>24</b>
<br />
```

### Steps to reproduce the behaviour
So far, I have this from the user:
```
I searched using the map and drew a box with Reading in the north-east corner and Sherborne St John in the south-west corner (just north of Basingstoke. 
```

**Comment by portableant** [comment]
Simple fix - add !is_null test to https://github.com/findsorguk/findsorguk/blob/master/app/modules/database/views/scripts/ajax/kml.phtml#L23

**Comment by s-moon** [comment]
Thanks @portableant

**Comment by portableant** [comment]
Closed in 4682e7ac6d5cc75803b78b6d2f7f55da6cb1587a
